gammapy/utils/fitting/sampling.py

```diff
@@ -0,0 +1,162 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# helper functions for mcmc smapling
+import numpy as np
+import emcee
+import matplotlib.pyplot as plt
```

Comment:
Matplotlib is an optional dependency. You have to put the import in the function(s) where you use it for Gammapy tests to pass (and for analysis to be possible without plotting)

Reply:
ok